package.json

```diff
@@ -64,5 +64,8 @@
   },
   "engines": {
     "node": ">=0.10.0"
+  },
+  "scripts": {
+    "postinstall": "bower install"
```

Comment:
`bower install` or `./node_modules/.bin/bower install`? I'd go with the latter and remove the requirement for developers to install bower globally. What do you think?